Eastern-Research-Group/mywaterway
# Bugfix/808 remove eq refs

## Related Issues:
* [HMW-808](https://jira.epa.gov/browse/HMW-808)
* [HMW-809](https://jira.epa.gov/browse/HMW-809)

## Main Changes:
* Removed unnecessary google analytics code from 404 and 500 pages.
  * I noticed this code was referencing expert query and uswds. 
* Updated the google analytics code to more accurately capture file/download link clicks.

## Steps To Test:
1. Verify the unnecessary GA code has been removed
2. Verify references to expert query and or owapps have been removed, excluding the services.json file.
3. Either turn on google analytics debugger or add `console.log('category: ', category);` to line 756 of `public\index.html`.
4. Click links in the app and verify the category is correct (i.e., Download if it opens a file, External if it opens a webpage outside of HMW and Internal if it opens a webpage in HMW). Below are some links that might be harder to find.
    * http://localhost:3000/waterbody-report/TDECWR/TN03150101012_0100/2020
    * http://localhost:3000/plan-summary/DOEE/59541
    * http://localhost:3000/state/FL/water-quality-overview
        * Expand documents
        * Verify none of the links end with `(UNKNOWN)`
        * Expand success stories
        * Verify none of the links end with `(UNKNOWN)`
